mmv1/products/cloudidentity/terraform.yaml

```diff
@@ -40,8 +40,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         test_env_vars:
           org_domain: :ORG_DOMAIN
           cust_id: :CUST_ID
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_identity_groups_dynamic"
```

Comment:
Ah- we _need_ to run these synchronously, due to some limitations on the API. We'll want to `skip_test: true` here, and "handwrite" the tests by copying the generated output from https://github.com/modular-magician/terraform-provider-google-beta/blob/c94e3f82c15502847b03b8eb89b7d6f34b897020/google-beta/resource_cloud_identity_group_generated_test.go#L26-L78 into https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/tests/resource_cloud_identity_group_test.go.erb, similar to what was done with `BasicExample`.

Reply:
(we failed to build because this injected the same piece of code twice, following up on why it's because we weren't generating tests prior to this change)

Reply:
I'm not understanding :(
Is there a way I can locally check whether what I've done works?

Reply:
Generating locally (https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-provider-tests-locally) + running (https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-provider-tests-locally) works, but an environment for these resources can be difficult to get set up. Just running `make build` in the provider directory will get you past the `terraform-provider-google-beta-test` error (as is the case here those are almost always build issues, not actually unit test issues)

Reply:
I think I got this done, could you take a look and let me know?

Thanks!